gnocchi/archive_policy.py

```diff
@@ -155,20 +158,25 @@ def __init__(self, granularity=None, points=None, timespan=None):
                 raise ValueError(
                     u"timespan ≠ granularity × points")
 
-        if granularity is not None and granularity <= 0:
-            raise ValueError("Granularity should be > 0")
+        if granularity is not None:
+            if not isinstance(granularity, numpy.timedelta64):
```

Comment:
Agreed. REST does the conversion but SQLAlchemy JSON type does not, it only unserialize the JSON and returns integer. So I can convert integer to Numpy type in SQLAlchemy but I'm not sure it's better than doing that here.